src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -259,4 +263,31 @@ default GameModeData getGameModeData() {
      */
     void setSleepingIgnored(boolean sleepingIgnored);
 
+    /**
+     * Gets a collection of {@link ServerBossBar}s this player is a member of.
+     *
+     * <p>The returned collection is unmodifiable.</p>
+     *
+     * @return The boss bars
+     */
+    Collection<ServerBossBar> getBossBars();
```

Comment:
Right, I'm not saying there should be, just saying it needs to be documented as such ;) I did poke around the internals, but I'm not really finding a reason to have a difference of `BossBar` and `ServerBossBar`. (Plus, there are entities that by default provide a `BossBar`, so getters could be added to those entities)